core/common/src/main/java/alluxio/network/protocol/databuffer/DataFileChannel.java

```diff
@@ -61,7 +61,7 @@ public void readBytes(byte[] dst, int dstIndex, int length) {
 
   @Override
   public int readableBytes() {
-    throw new UnsupportedOperationException("DataFileChannel#readableBytes is not implemented.");
+    return (int) mLength;
```

Comment:
Is this true, since this cannot be read directly?

Reply:
reverted.